libraries/utils/css/breadcrumbs.css

```diff
@@ -0,0 +1,24 @@
+.breadcrumbs {
```

Comment:
More of a discussion opener than a direct suggestion: isn't the class name too generic? We typically use prefixes, such as `ui`, `d4`, or shortened package names.